src/java.base/windows/classes/sun/nio/fs/WindowsFileSystemProvider.java

```diff
@@ -47,6 +47,7 @@
     extends AbstractFileSystemProvider
 {
     private static final Unsafe unsafe = Unsafe.getUnsafe();
+    private static final byte[] emptyPath = new byte[0];
```

Comment:
Is "unsafe" used, maybe this could be removed while you are there.
Probably should be EMPTY_PATH to be consistent with the name in UnixFileSystemProvider.